automation-roles/40-configure-infra/openshift-cloud-infra/vars/main.yml

```diff
@@ -0,0 +1,9 @@
+existing_ocp_cloud_infra:
+  ibm-roks:
+    storage_class: "ibmc-vpc"
```

Comment:
This is not the correct storage class for classic clusters. Let me check which ones are always available.

Reply:
@fketelaars , in fact, this is storage class filter.